WordPress/src/main/java/org/wordpress/android/ui/mediapicker/MediaPickerFragment.kt

```diff
@@ -379,8 +382,29 @@ class MediaPickerFragment : Fragment() {
                 recycler.visibility = View.VISIBLE
                 setupAdapter(uiModel.items)
             }
-            Empty -> {
+            is Empty -> {
                 actionable_empty_view.visibility = View.VISIBLE
+                actionable_empty_view.title.text = uiHelpers.getTextOfUiString(requireContext(), uiModel.title)
+                if (uiModel.htmlSubtitle != null) {
+                    actionable_empty_view.subtitle.text = Html.fromHtml(
+                            uiHelpers.getTextOfUiString(
+                                    requireContext(),
+                                    uiModel.htmlSubtitle
+                            ).toString()
+                    )
+                    actionable_empty_view.subtitle.movementMethod = WPLinkMovementMethod.getInstance()
+                    actionable_empty_view.subtitle.visibility = View.VISIBLE
+                } else {
+                    actionable_empty_view.subtitle.visibility = View.GONE
+                }
+
+                if (uiModel.image != null) {
+                    actionable_empty_view.image.setImageResource(uiModel.image)
+                    actionable_empty_view.image.visibility = View.VISIBLE
+                } else {
+                    actionable_empty_view.image.visibility = View.GONE
+                }
+                actionable_empty_view.updateLayoutForSearch(uiModel.isSearching, 0)
```

Comment:
hi @planarvoid, I tested this again but I'm now getting empty state without image in Stock media. Tried to move the `updateLayoutForSearch` call right after the `is Empty -> {` block and it worked; so probably that call effect is dependent on call order. Let me know wdyt 🙂 .

| From this PR | Moving the updateLayoutForSearch call at the beginning of the block |
|---|---|
| ![image](https://user-images.githubusercontent.com/47797566/95797501-0033a100-0cf0-11eb-8828-d835ee9f1cbc.png) | ![image](https://user-images.githubusercontent.com/47797566/95797410-c2367d00-0cef-11eb-80cf-dcf7020118d5.png) |

Reply:
good catch @develric, it's fixed in 77619bab3eea81e483bb896c43bf96217f844d59